File: C11_i18n/C11_i18n/ViewController.m
//
//  ViewController.m
//  C11_i18n
//
//  Created by ucom Apple 13 on 2016/12/2.
//  Copyright © 2016年 Gillian_studio. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *uilabel;
@property (weak, nonatomic) IBOutlet UILabel *msglabel;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    self.uilabel.text = NSLocalizedString(@"LABEL_COMPANY", nil);
    self.msglabel.text = NSLocalizedStringFromTable(@"MSG_GREETING", @"Messages", nil );
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: L10/L10/ViewController.swift
//
//  ViewController.swift
//  L10
//
//  Created by ucom Apple 13 on 2016/12/2.
//  Copyright © 2016年 Gillian_studio. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var comfirmHandler: UIButton!
    @IBOutlet weak var txtMsg: UITextField!
    @IBOutlet weak var timePicker: UIDatePicker!
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    @IBAction func btncomfirmHandler(sender: AnyObject) {
        
        self.txtMsg.resignFirstResponder() // close keyboard
        
        var localNotification : UILocalNotification = UILocalNotification() ;
        
        localNotification.fireDate = self.timePicker.date;
        localNotification.timeZone = NSTimeZone.defaultTimeZone()
        localNotification.alertBody = self.txtMsg.text ;
        
        localNotification.soundName = UILocalNotificationDefaultSoundName;
        localNotification.applicationIconBadgeNumber = UIApplication.sharedApplication().applicationIconBadgeNumber + 1 ;
        
         UIApplication.sharedApplication().scheduleLocalNotification(localNotification)
//        UIApplication.sharedApplication().scheduledLocalNotifications( localNotification ) ;
        
        print("Done");
    }
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

